### ISSUE TITLE
Does it make sense to leverage Cloudflare's DNS 1.1.1.3 somehow?

### ISSUE THREAD

**Issue by wingman-jr-addon** [wingman-jr-addon/wingman_jr#53]
https://blog.cloudflare.com/introducing-1-1-1-1-for-families/

Perhaps this could be used as a partial stopgap for video? Or some other usage as a blacklist?

**Comment by wingman-jr-addon** [comment]
Think it might be possible using the DNS over HTTPS provided by Cloudflare. See [here for API example](https://developers.cloudflare.com/1.1.1.1/dns-over-https/json-format/) and [here for the endpoint](https://developers.cloudflare.com/1.1.1.1/1.1.1.1-for-families/setup-instructions/dns-over-https/)
Note I need to try the family endpoint to make sure it supports the same query.

**Comment by wingman-jr-addon** [comment]
I tried this and it does seem to work. For example, here's the normal resolver against a known bad site:
```
> curl -H "accept: application/dns-json" "https://cloudflare-dns.com/dns-query?name=pornhub.com&type=AAAA"
{"Status": 0,"TC": false,"RD": true, "RA": true, "AD": false,"CD": false,"Question":[{"name": "pornhub.com.", "type": 28}],"Authority":[{"name": "pornhub.com.", "type": 6, "TTL": 2531, "data": "dns1.p06.nsone.net. hostmaster.nsone.net. 1581001896 43200 7200 1209600 3600"}]}
```
Now, if we check against the family domain:
```
> curl -H "accept: application/dns-json" "https://family.cloudflare-dns.com/dns-query?name=pornhub.com&type=AAAA"
{"Status":0,"TC":false,"RD":true,"RA":true,"AD":false,"CD":false,"Question":[{"name":"pornhub.com","type":28}],"Answer":[{"name":"pornhub.com","type":28,"TTL":60,"data":"::"}]}
```

**Comment by wingman-jr-addon** [comment]
Released in 1.1.0
